wasm/validator/derive/src/entrypoint.rs

```diff
@@ -31,10 +28,7 @@ impl syn::parse::Parse for Attr {
 #[derive(PartialEq, Eq)]
 enum ParamType {
     Authority,
-    Transaction,
-    Instruction,
-    Query,
-    Expression,
```

Comment:
why would you remove this?

Reply:
Because now we have only one validator. And it should accept `NeedsPermissionBox`, so it should support validation of any of 3 variants